visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -94,11 +94,31 @@ export class CustomConfigHelper {
 	}
 
 	private static loadCustomConfigs() {
+		const ccLocalStorage = this.getCcLocalStorage()
+		return new Map(ccLocalStorage?.customConfigs)
+	}
+
+	// TODO [2022-08-01]: remove replace method for SINGLE mode before when deadline is reached
+	private static getCcLocalStorage() {
 		const ccLocalStorage: LocalStorageCustomConfigs = JSON.parse(
 			CustomConfigHelper.getStorage().getItem(CUSTOM_CONFIGS_LOCAL_STORAGE_ELEMENT),
 			stateObjectReviver
 		)
-		return new Map(ccLocalStorage?.customConfigs)
+
+		if (localStorage?.getItem("codeChartaVersion") < "1.88.0") {
+			for (const [, customConfig] of ccLocalStorage.customConfigs.values()) {
+				if (customConfig.mapSelectionMode === "SINGLE") {
+					customConfig.mapSelectionMode = CustomConfigMapSelectionMode.MULTIPLE
+				}
+			}
+			this.replaceSingleModeInLocalStorage(ccLocalStorage)
+		}
```

Comment:
- I think this belongs to`replaceSingleModeInLocalStorage` as such I would move it into it.
- There is a posibility that ccLocalStorage.customConfigs.values() crashes, if no customConfigs are present.